kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -543,7 +543,7 @@ define([
                                 );
                             });
                         }
-                        this.bus.emit(jcm.MESSAGE_TYPE.START_UPDATE, {
+                        this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
```

Comment:
remove all the `START_UPDATE` calls and change them to generic `STATUS` calls. The `STOP_UPDATE` calls can be removed completely.